sys-devel/gcc/files/gcc-12.1.0-recognize-mmacosx-version-min-13.0-and-newer.patch

```diff
@@ -0,0 +1,50 @@
+From 6725f186cb70d48338f69456864bf469a12ee5be Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link to the upstream commit and any other useful references at the top of the patch.